### DISCUSSION TITLE
评测结果内容为空

### DISCUSSION THREAD

**Discussion by yingwen330** [open-compass/opencompass#433] [Q&A] [answered]
脚本如下：  hf_vicuna_33b_v13.py
from opencompass.models import HuggingFaceCausalLM

models = [
    dict(
        type=HuggingFaceCausalLM,
        abbr='vicuna-33b-v1.3-hf',
        path="/data/models/vicuna-33b-v1.3",
        tokenizer_path="/data/models/vicuna-33b-v1.3",
        tokenizer_kwargs=dict(
            padding_side='left',
            truncation_side='left',
            use_fast=False,
        ),
        max_out_len=100,
        max_seq_len=2048,
        batch_size=8,
        model_kwargs=dict(device_map='auto'),
        batch_padding=False, # if false, inference with for-loop without batch padding
        run_cfg=dict(num_gpus=4, num_procs=1)
    )
]

执行脚本命令:python run.py --models hf_vicuna_33b_v13 --datasets ceval_gen

结果如下：
![image](https://github.com/open-compass/opencompass/assets/131145127/6e5056f2-aef2-4007-8c1e-8d18dac30cfd)

**Comment by SefaZeng** [comment]
同样的问题，请问解决了吗？

**Reply by mmarzl17** [reply]
解决了吗

**Reply by 254038lqq** [reply]
您好，请问问题解决了吗？

**Comment by Leymore** [comment]
The previous log should have a hint similar to the one below. You can open and view this `.out` file to see the error message.

```text
11/01 14:39:00 - OpenCompass - WARNING - task OpenICLInfer[InternLM-20B-Chat/math_19] failed, see                                                                              | 20/993 [07:03<3:41:23, 13.65s/it]
./outputs/20231031_regression/chat/20231031_225246/logs/infer/InternLM-20B-Chat/math_19.out
```

**Comment by SijunWang** [comment]
'torchrun' 不是内部或外部命令，也不是可运行的程序或批处理文件。这个怎么解决？

**Comment by Leymore** [comment, marked as answer]
install torchrun via `pip3 install torch`. 
You may need to follow instruction from https://opencompass.readthedocs.io/en/latest/get_started/installation.html
